### PR TITLE
Add recommendation for installing ezplatformsearch & richtext-datatype-bundle

### PR DESCRIPTION
**Pull request by andrerom**

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -108,6 +108,17 @@ _Tip:_
 composer require --update-no-dev "ezsystems/legacy-bridge:^2.0"
 ```
 
+### Recommended: Add addtional Legacy <=> eZ Platform integrations
```
Comment (emodric) [inline INSTALL.md]: `addtional` => `additional`

File: INSTALL.md [hunk 2]
```diff
@@ -108,6 +108,17 @@ _Tip:_
...
+
+**[netgen/ezplatformsearch](https://github.com/netgen/ezplatformsearch)**
+
+When using either Solr or _(future)_ Elastic search bundle in eZ Platform, this eZ Publish extenion takes care of integrating with eZ Platform Search API, and among other things makes sure updates in legacy triggers search indexing updates, and all out of the box search featurs in eZ Publihs uses the same search index.
```
Comment (emodric) [inline INSTALL.md]: `extenion` => `extension`
`triggers` => `trigger`

Also, how about `This also makes sure eZ Platform and eZ Publish Legacy use the same search index.` ?

File: INSTALL.md [hunk 3]
```diff
@@ -108,6 +108,17 @@ _Tip:_
...
+
+**[netgen/richtext-datatype-bundle](https://github.com/netgen/NetgenRichTextDataTypeBundle)**
+
+If you plan to migrate some or all of your content types to eZ Platform RichText format, install this bundle which extends legacy so you can edit content also with richtext field types using raw XML editing text box. Good for cases where content migration, and also backend migration needs to happen gradually over time.
```
Comment (emodric) [inline INSTALL.md]: How about: `If you plan to migrate some or all of your content types to eZ Platform RichText format, install this bundle to make it possible to edit content with RichText field types using raw XML editing text box in eZ Publish Legacy. Good for cases...`

**Comment by andrerom** [comment]
@emodric fixed++ :)

**Comment by emodric** [comment]
@andrerom 

> for which is then used for search

First *for* in here is not needed. Other than that, :+1:

**Comment by andrerom** [comment]
@emodric fixed

**Comment by emodric** [comment]
Merge away!

**Pull request merged**